src/index.js

```diff
@@ -1,14 +1,45 @@
-import _ from 'lodash';
 import './style.css';
+import { addListeners, dragOver } from './drag.js';
 
-function component() {
-    const element = document.createElement('div');
-  
-    // Lodash, now imported by this script
-    element.innerHTML = _.join(['Hello', 'webpack'], ' ');
-    element.classList.add('hello');
-  
-    return element;
-  }
-  
-  document.body.appendChild(component());
\ No newline at end of file
+const todoList = document.querySelector('.todoList');
+
+const items = document.querySelectorAll('.item');
+
+const list = [
+  {
+    index: 1,
+    description: 'brush my teeth',
+    completed: true,
+  },
+  {
+    index: 2,
+    description: 'Go to school',
+    completed: false,
+  },
+  {
+    index: 3,
+    description: 'Sleep',
+    completed: false,
+  },
+];
```

Comment:
As mentioned by the last reviewer, there are only 3 items on the list, but each item is rendered twice. Kindly fix the bug.